cmd/axiom/auth/auth_login.go

```diff
@@ -133,8 +133,9 @@ func completeLogin(opts *loginOptions) error {
 		); err != nil {
 			return err
 		}
-		if deploymentKind == typeCloud {
-			opts.URL = axiom.CloudURL
```

Comment:
Don't we still need this?

Reply:
This seemed to be a copy&paste error. The user is asked for his deployment type and only in case of self host, for the url. This check is already present after the previous question in line 122.